src/components/CheckBox.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+const CheckBox = props => {
+  const { onChange, checked } = props;
+
+  return (
+    <React.Fragment>
+      <div className="container">
+        <div className="flexItem">
+          <div className="dg_checkbox">
+            <input
+              name="popularity"
+              id="popularity-color-black"
+              value={checked}
+              className="dg_checkbox-input"
+              type="checkbox"
+              onChange={onChange}
+            />
+            <label
+              className="dg_label dg_checkbox-label"
+              htmlFor="popularity-color-black"
+            >
+              POPULARITY
```

Comment:
![image](https://user-images.githubusercontent.com/1933400/72902735-ff8a5300-3cf9-11ea-9ff9-d20fc3438fd9.png)

I think this should be something like `Show Popular Services` or something along those lines, need to check with web services.

Reply:
done